src/lib/fees/model.ts

```diff
@@ -108,6 +124,16 @@ export const calculateFees: CalculateFeeBps = ({ tradeAmountUsd, foxHeld, feeMod
   const feeUsd = feeUsdBeforeDiscount.minus(feeUsdDiscount)
   const foxDiscountUsd = feeUsdBeforeDiscount.times(foxDiscountPercent.div(100))
 
+  console.log({
+    feeBps: feeBps.toFixed(),
+    feeBpsFloat: feeBpsFloat.toFixed(),
+    feeUsd: feeUsd.toFixed(),
+    foxDiscountPercent: foxDiscountPercent.toFixed(),
+    foxDiscountUsd: foxDiscountUsd.toFixed(),
+    feeUsdBeforeDiscount: feeUsdBeforeDiscount.toFixed(),
+    feeBpsBeforeDiscount: feeBpsBeforeDiscount.toFixed(),
+  })
+
```

Comment:
nitpick:

```suggestion
```

Reply:
fml, changed in [30b7cb9](https://github.com/shapeshift/web/pull/8044/commits/30b7cb926fe202f7e1e9e44fd5a3616439f2f4f9)